sentinel-core/src/main/java/com/alibaba/csp/sentinel/event/subscriber/EventSubscriber.java

```diff
@@ -0,0 +1,31 @@
+/*
+ * Copyright 1999-2018 Alibaba Group Holding Ltd.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.alibaba.csp.sentinel.event.subscriber;
+
+import com.alibaba.csp.sentinel.event.Event;
+
+/**
+ * @author lianglin
+ * @since 1.7.0
+ */
+public interface EventSubscriber {
+    /**
+     * Subscribe the event and make some  business logic processing
+     *
+     * @param event
+     */
+    void listen(Event event);
```

Comment:
For the circuit breaker, maybe it's more clear to be `onOpen` and `onClose`?

Reply:
If I want to define a general subscription interface, it can be used for more than circuit breaker, such as flow limiting event or other events. Any suggestions?